biblio-dblp.el

```diff
@@ -30,16 +30,30 @@
 
 (require 'biblio-core)
 
+(defcustom biblio-dblp-preferred-format 'standard
+  "Format of the Bibtex entry returned by DBLP.
+Can be one of `standard', `condensed', or `with-crossref' (default is `standard')"
+  :type '(radio (const standard)
+                (const condensed)
+                (const with-crossref)))
```

Comment:
Should we document what each of these options do?  `defcustom` lets you replace each with a string instead of the name of the symbol in the UI.